nanoFramework.System.IO.FileSystem.nuspec

```diff
@@ -32,7 +33,7 @@ This package requires a target with System.IO.FileSystem v$nativeVersion$ (check
     <file src="System.IO.FileSystem\bin\Release\System.IO.FileSystem.xml" target="lib\System.IO.FileSystem.xml" />
     <file src="assets\readme.txt" target="" />
     <file src="README.md" target="docs\" />
-    <file src="assets\nf-logo.png" target="images\" />
+    <file src="assets\nf-logo.png" target="images\nf-logo.png" />
```

Comment:
There is no need to specify the destination filename, unless you want to change the file name.